src/index.js

```diff
@@ -1,3 +1,5 @@
+import 'src/outdated-browser-message'
```

Comment:
My understanding was that `outdated-browser-message` was a separate entry point so that it would be loaded even if an older browser failed to parse the main bundle. But since we already split off most of the application into a separate chunk by dynamically importing `appLoader`, this seems like it would be fine as well.

Reply:
That's what I figured, yeah...